apollo-router/src/uplink/testdata/connectv0_3.router.yaml

```diff
@@ -5,4 +5,4 @@ homepage:
 limits:
   parser_max_recursion: 1000
 connectors:
-  preview_connect_v0_2: true
+  preview_connect_v0_3: true
```

Comment:
So could we totally omit this `connectors` section now, and we'd get `connect/v0.2` support?

Reply:
correct, this was just a gate we added so we could merge code to dev/release router versions without allowing users to accidentally use 0.2 features. we're removing the gate when we're officially releasing 0.2.